server/src/main/java/io/spine/server/entity/Transaction.java

```diff
@@ -444,6 +439,18 @@ public void setDeleted(boolean deleted) {
     }
 
     /**
+     * Retrieves the {@link EntityVersioning} of current {@code Transaction}.
+     *
+     * <p>The value should be constant among all the instances of a certain (runtime) type.
+     *
+     * <p>The default value is the Event Sourcing-fashioned
```

Comment:
`By default the version is taken from the {@linkplain ... latest event applied}`.